### ISSUE TITLE
Performance Degradation on Large Pages Continues

### ISSUE THREAD

**Issue by carlossierra311** [BookStackApp/BookStack#4100]
### Attempted Debugging

- [X] I have read the debugging page

### Searched GitHub Issues

- [X] I have searched GitHub for the issue.

### Describe the Scenario

After installing v23.02, which addressed bug #3981, the performance degradation continues to affect me badly on large pages, to the point of it still not being usable. I still get delays of 10, 20 and more seconds after writing a few words. 

I would appreciate any additional help you can provide. Thanks in advance!  

### Exact BookStack Version

v23.02

### PHP Version

8.1.10

### Hosting Environment

Windows Server 2019 Standard v1809, OS build 17763.3406
32 GB Ram, Intel Xeon E3-1230 v5 @3.4GHz

BookStack installed manually by following [these steps](https://www.bookstackapp.com/docs/admin/installation/#manual), using the following components:
php-8.1.10
mysql-8.0.30
nginx-1.23.1

**Comment by ssddanbrown** [comment]
Hi @carlossierra311,

> I still get delays of 10, 20 and more seconds after writing a few words.

Okay, is this a different form of lag/delay? Before it sounded like it was upon first type. Have things improved since v23.02?

Again, it would help to have a reproduction sample of the content being edited that causes this, otherwise I'm somewhat blind. I tried using the World-war-2 wikipedia page you advised to use last time, doubled it, added a few code blocks, but still could not reproduce. 

As before, there are fundamental reasonable limits that could be reached, which I can inspect to assess reasonability & look for improvements within, if I can specifically debug the scenario but I need something to go on.

Something else to check is to give things a test in a different browser or incognito window. Some plugins can do strange things when it comes to inputs and large amounts of data.

**Comment by carlossierra311** [comment]
Hi @ssddanbrown. You are totally right. Editing the page on Incognito Mode solved the lag. I then disabled the plugins I have in Chrome, and edited on a Regular Mode tab, and the lag was gone too. Finally, I was able to single out the Microsoft Editor plugin, which was the culprit of the bad performance. Now I can edit without any lag whatsoever. 
Thanks for your support. I really appreciate it!
